hazelcast/src/test/java/com/hazelcast/nio/tcp/iobalancer/IOBalancerStressTest.java

```diff
@@ -53,6 +61,7 @@ public void killAllHazelcastInstances() throws IOException {
         HazelcastInstanceFactory.terminateAll();
     }
 
+    @Repeat(25)
```

Comment:
It is a nightly tests, so intended to run for some time to make sure it doesn't fail.